go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pulumi/pulumi-terraform-bridge/v3
 
-go 1.23.11
+go 1.24.0
```

Comment:
Why the bump? I thought we follow the same policy as pu/pu and try to stay 2 releases behind latest.

Reply:
Yeah, in pulumi/pulumi we keep the older version as long as we don't see a good reason to update (as long as we don't want to use any new features from the newer release). Imo providers should not be bound by `pulumi/pulumi` Go version upgrades here, and just support the last 2 releases as is convenient.

Reply:
I think this was resolved via rebase onto the changes from #3191.